### PR TITLE
openimageio: unbreak musl

### PR DESCRIPTION
**Pull request by pullmoll**

### CODE REVIEW

**Comment by beefcurtains** [comment]
:+1:

File: srcpkgs/openimageio/template [hunk 1]
```diff
@@ -20,6 +20,9 @@ distfiles="https://github.com/OpenImageIO/oiio/archive/Release-${version}.tar.gz
 checksum=497c60bba9525b913935f0af771126338f07c2f78bd84e96fd19df38b6574706
 nocross=yes # -> ilmbase
 
+pre_build() {
+	find ${wrksrc} -type f -exec sed -i "{}" -e "s;<sys/poll\.h>;<poll.h>;g" \;
```
Comment (Gottox) [inline srcpkgs/openimageio/template]: whats the ```{}``` good for?

Comment (pullmoll) [inline srcpkgs/openimageio/template]: The `{}` is replaced with the current filename with `-exec`.

Comment (Gottox) [inline srcpkgs/openimageio/template]: Ah, I'm blind.

**Pull request merged**